avd_docs/aws/iam/AVD-AWS-0342/docs.md

```diff
@@ -1,10 +1,8 @@
 
-In iam:PassRole the service carrying out the actions is "provided" a role by the calling principal and implicitly takes on that role to carry out the actions (instead of executing sts:AssumeRole).
-       The privileges attached to the role are distinct from those of the primary ordering the action and may even be larger and can cause security issues.
-           
+Ensures any IAM pass role attched to roles are flagged and warned.
 
 ### Impact
-Compromise on security of aws resources.
+<!-- Add Impact here -->
```

Comment:
ditto

Reply:
It was part of aws/iampassrole_policy branch don't know why this is keep appearing again and again.